docs/public-networks/how-to/configure-besu/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Configure Besu
+description: Specify options in the Besu configuration file.
+sidebar_position: 1
+tags:
+  - public networks
+  - private networks
+---
+
+# Configure Besu
+
+Besu comes with a [default configuration](#default-configuration) that is suitable for staking.
+
+You can override the default values by specifying [configuration options](../../reference/cli/options.md) on the command line, as environment variables, or in a [TOML configuration file](#toml-configuration-file) that can be reused across node startups.
+
+You can also use a [pre-configured profile](profile.md) for some common use cases or create and apply a [custom profile](profile.md#load-external-profiles).
```

Comment:
@joaniefromtheblock I did (final?!) update to improve visibility/connection to the item you added PTAL so I am not the only eyes on this -- If ok here

Reply:
then, please merge